flow/packet-headers/snmpv2.yaml

```diff
@@ -0,0 +1,308 @@
+          type: string
+          default: no_error
+          x-enum:
+            no_error:
+              x-field-uid: 1
+            too_big:
+              x-field-uid: 2
+            no_such_name:
+              x-field-uid: 3
+            bad_value:
+              x-field-uid: 4
+            read_only:
+              x-field-uid: 5
+            gen_err:
+              x-field-uid: 6
+            no_access:
+              x-field-uid: 7
+            wrong_type:
+              x-field-uid: 8
+            wrong_length:
+              x-field-uid: 9
+            wrong_encoding:
+              x-field-uid: 10
+            wrong_value:
+              x-field-uid: 11
+            no_creation:
+              x-field-uid: 12
+            inconsistent_value:
+              x-field-uid: 13
+            resource_unavailable:
+              x-field-uid: 14
+            commit_failed:
+              x-field-uid: 15
+            undo_failed:
+              x-field-uid: 16
+            authorization_error:
+              x-field-uid: 17
+            not_writable:
+              x-field-uid: 18
+            inconsistent_name:
+              x-field-uid: 19
+          x-field-uid: 2
+        error_index:
+          x-field-pattern:
+            description: >-
+              When Error Status is non-zero, 
+              this field contains a pointer that specifies which object generated the error. 
+              Always zero in a request.
+            format: integer
+            length: 32
+            default: 0
+            features: [count]
+          x-field-uid: 3
+        variable_bindings: 
+          description: >-
+            A Sequence of variable_bindings.
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.Snmpv2.VariableBinding'
+          x-field-uid: 4
+    Flow.Snmpv2.BulkPDU:
+      description: The purpose of the GetBulkRequest-PDU is to request
+          the transfer of a potentially large amount of data, including, but
+          not limited to, the efficient and rapid retrieval of large tables.
+      type: object
+      properties:
+        request_id:
+          x-field-pattern:
+            description: >-
+                Identifies a particular SNMP request. 
+                This index is echoed back in the response from the SNMP agent, 
+                allowing the SNMP manager to match an incoming response to the appropriate request.
+            format: integer
+            length: 32
+            default: 0
+            signed: true
+            features: [count]
+          x-field-uid: 1
+        non_repeaters:
+          x-field-pattern:
+            description: >-
+                One variable binding in the Response-PDU is requested for the first non_repeaters variable bindings in the GetBulkRequest.
+            format: integer
+            length: 32
+            default: 0
```

Comment:
Any specific reason to skip the counter (increment/decrement) pattern for this field?

Reply:
@PrasenjitAdhikary : Having this as variable basically means that the different packets in the flow would normally have different sizes due to different number of OIDs .. where no. of OIDs ( variable bindings ) >= value in this field. I think it is not possible to do that inside a single flow and will be difficult to actually use to create a flow with multiple valid packets with this field having different values ( similar to say trying to do this with length field in IP packets ) ... Just ability to put any value in this field should also cover negative use-cases .

Reply:
Yes true, that is the logic behind not having counter on this field, @PrasenjitAdhikary